### PR TITLE
survival_cif documents added

### PR DESCRIPTION
**Pull request by clyau**

### CODE REVIEW

File: R/survival_cif.qmd [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+## R Packages
+
+We identify three packages:
+
+-   `cmprsk` version 2.2-12, release date 2024-05-14
+
+-   `tidycmprsk` version 1.0.0, release date 2023-10-30
+
+-   `survival` version 3.7-0, release date 2024-06-01
```
Comment (orladoylenvs) [inline R/survival_cif.qmd:21]: @statasaurus  - I guess we want to handle this via renv and remove the versions and release dates? What do you think?

File: data/bmt.sas7bdat
Comment (orladoylenvs) [inline data/bmt.sas7bdat:1]: @statasaurus assuming we wouldn't include this in the repo given we don't run the sas code?

Comment (clyau) [reply data/bmt.sas7bdat:1]: just a note: the data/bmt.sas7bdat is used by R/survival_cif.qmd

File: R/survival_cif.qmd (hunk 1 above)
Comment (statasaurus) [reply R/survival_cif.qmd:21]: I realised this yesterday. I am going to update the template you made. But if you want to have session info please follow the code from [here](https://github.com/PSIAIMS/CAMIS/issues/139)

File: renv.lock
[the remark this answers is not in the document]
Comment (LucaAnholt) [reply renv.lock:1]: I opened a PR to fix the renv conflicts https://github.com/clyau/CAMIS/pull/5 but it looks like @statasaurus also fixed this, so no need to accept this PR anymore @clyau!

[the remark this answers is not in the document]
Comment (statasaurus) [reply renv.lock:1]: So sorry I didn't realise you were on it @LucaAnholt! Thank you

**Pull request merged**